### PR TITLE
feat: Generate CSS files from Figma using Style Dictionary

### PR DESCRIPTION
**Pull request by tft-abel**
We generate CSS files for all organizations directly from Figma to the `theme-fs` folders. The original files are left untouched.

When merged, this will be published as `11.1.0-alpha.0` such that it can be used for testing in https://github.com/AtB-AS/kundevendt/issues/19404.

### CODE REVIEW

**Review by gorandalum — APPROVED** [review]
Looks nice!

File: packages/theme/src/generated/themes-fs/atb-theme/dark.css [hunk 1]
```diff
@@ -0,0 +1,755 @@
...
+  --typography-web-number: -0.5;
+  --spacing-x-small: 4;
+  --spacing-small: 8;
+  --spacing-medium: 12;
+  --spacing-large: 20;
+  --spacing-x-large: 24;
+  --icon-size-x-small: 12;
+  --icon-size-small: 16;
+  --icon-size-normal: 20;
+  --icon-size-large: 26;
```
Comment (rosvik) [inline packages/theme/src/generated/themes-fs/atb-theme/dark.css:754]: Have you considered how we will add units to these values? Today we have e.g. `--border-radius-small: 0.25rem;`

Comment (tft-abel) [reply packages/theme/src/generated/themes-fs/atb-theme/dark.css]: Great catch! The type of these variables was not configured correctly in Style Dictionary. Fixed that now and added the `size/pxToRem` transformer in Style Dictionary. Should be correct now

**Review by rosvik — APPROVED** [review]
💯

**Comment by tft-abel** [comment]
Testing is part of the usage of this in webshop and journey planner

**Pull request merged**